### PR TITLE
Xp addition

### PR DESCRIPTION
**Pull request by MrNinja008**

### CODE REVIEW

File: BasicScore/src/Ifera/BasicScore/Main.php [hunk 1]
```diff
@@ -37,6 +37,7 @@ public function onEnable(){
 
 			foreach($this->getServer()->getOnlinePlayers() as $player){
 				(new PlayerTagUpdateEvent($player, new ScoreTag("basicscore.ping", strval($player->getPing()))))->call();
+                                (new PlayerTagUpdateEvent($player, new ScoreTag("basicscore.xp", strval($player->getXpLevel()))))->call();
```
Comment (Ifera) [inline BasicScore/src/Ifera/BasicScore/Main.php:40]: Bad formatting

File: BasicScore/src/Ifera/BasicScore/listeners/EventListener.php [hunk 2]
```diff
@@ -79,6 +80,14 @@ public function onTeleport(EntityTeleportEvent $event){
...
 		(new PlayerTagUpdateEvent($player, new ScoreTag("basicscore.world_folder_name", $target->getFolderName())))->call();
 	}
+        
+        public function onExperienceChange(PlayerExperienceChangeEvent $event){
+               $player = $event->getPlayer();
+               $newLevel = $event->getXpLevel();
+    
+              (new PlayerTagUpdateEvent($player, new ScoreTag("basicscore.xp", $newLevel->getXpLevel())))->call();
+     
+        }
```
Comment (Ifera) [inline BasicScore/src/Ifera/BasicScore/listeners/EventListener.php:90]: Same as above, the formatting is all over the place.

File: BasicScore/src/Ifera/BasicScore/listeners/TagResolveListener.php [hunk 3]
```diff
@@ -114,6 +114,9 @@ public function onTagResolve(TagsResolveEvent $event){
 			case "date":
 				$value = date(ScoreHudSettings::getDateFormat());
 			break;
+                        
+                        case "xp":
```
Comment (Ifera) [inline BasicScore/src/Ifera/BasicScore/listeners/TagResolveListener.php:118]: bad formatting again

File: BasicScore/src/Ifera/BasicScore/listeners/TagResolveListener.php [hunk 4]
```diff
@@ -114,6 +114,9 @@ public function onTagResolve(TagsResolveEvent $event){
...
+                                $value = $player->getXpLevel();
```
Comment (Ifera) [inline BasicScore/src/Ifera/BasicScore/listeners/TagResolveListener.php:120]: missing `break` statement.

File: BasicScore/README.md [hunk 5]
```diff
@@ -4,6 +4,7 @@
 |:--:|:--:|
 |`{basicscore.name}`, `{basicscore.real_name}`|The name of the player|
 |`{basicscore.display_name}`|Display name of the player|
+|`{basicscore.xp}`|Display Xp Level Of Player|
```
Comment (Ifera) [inline BasicScore/README.md:7]: Don't capitalize the starting letter of every word.

**Review by Ifera — CHANGES_REQUESTED** [review]
Please do the changes requested. The PR will only be merged once all changes have been done.

**Comment by MrNinja008** [comment]
Sloved Everything lol

**Comment by Ifera** [comment]
Superseded by #16

**Pull request closed without merging**